pkgs/top-level/aliases.nix

```diff
@@ -647,6 +647,9 @@ mapAliases ({
   pg_hll = postgresqlPackages.pg_hll;
   pg_cron = postgresqlPackages.pg_cron;
   pg_topn = postgresqlPackages.pg_topn;
+  pinentry =
+    pinentryFlavors.all // pinentryFlavors
+    // { flavors = lib.attrNames pinentryFlavors; }; # added 2021-08-11
```

Comment:
Can't we use `pinentryFlavors.all` here?

Reply:
Then it wouldn't be compatible to the previous interface unfortunately. This should provide all flavors and outputs for each flavors and a passthrough for all flavors that were built. :/